refinery/ui/source/js/dashboard/controllers/dashboard.js

```diff
@@ -1136,24 +1139,29 @@ DashboardCtrl.prototype.getOriginalUri = function (eventData) {
  * Turns an unreadable date into a readable date string.
  *
  * @method  readableDate
- * @author  Fritz Lekschas
- * @date    2016-05-09
- * @param   {Object}  dataSet   Data set object of interest.
+ * @author  Fritz Lekschas & Scott Ouellette
+ * @date    2016-09-30
+ * @param   {Object}  object   DataSet or Analysis object of interest.
  * @param   {String}  property  Name of the date property to be made readable.
  * @return  {String}            Readable date string.
  */
-DashboardCtrl.prototype.readableDate = function (dataSet, property) {
+DashboardCtrl.prototype.readableDate = function (object, property) {
```

Comment:
avoid using the object name, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object

dataObj ???